client/coral-framework/components/FormField.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import styles from './styles.css';
+
+const FormField = ({className, showErrors = false, errorMsg, label, ...props}) => (
```

Comment:
you could probably get away with deleting this altogether if you used the `coral-ui` one